webapp/src/Controller/Jury/SubmissionController.php

```diff
@@ -1129,4 +1129,15 @@ protected function processClaim($judging, Request $request, ?string &$claimWarni
 
         return null;
     }
+
+    /**
+     * @Route("/{submitId<\d+>}/create-tasks", name="jury_submission_create_tasks")
+     */
+    public function createJudgeTasks(
+        string $submitId
+    ): RedirectResponse {
+        $this->dj->unblockJudgeTasksForSubmission($submitId);
+        $this->addFlash('info', "Started judging for submission: $submitId");
+        return $this->redirectToRoute('jury_submissions');
```

Comment:
I would think this should redirect to the submission itself since you specifically requested it, right?